### PR TITLE
refactor: remove duplicate object key in common.json

### LINKED ISSUE #15625: Remove duplicate object keys
Found a bug? Please fill out the sections below. 👍

### Issue Summary

On this file:
apps\web\public\static\locales\en\common.json duplicate keys

duplicate_keys

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Actual Results

- What's happening right now that is different from what is expected

### Expected Results

- This is an ideal result that the system should get after the tests are performed

### Technical details

- Browser version, screen recording, console logs, network requests: You can make a recording with [Bird Eats Bug](https://birdeatsbug.com/).
- Node.js version
- Anything else that you think could be an issue.

### Evidence

- How was this tested? This is quite mandatory in terms of bugs. Providing evidence of your testing with screenshots or/and videos is an amazing way to prove the bug and a troubleshooting chance to find the solution.

### PR DESCRIPTION
**Pull request by RushikeshGandhmal**
## What does this PR do?

- Fixes #15625 

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A-I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [x] N/A-I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

**Review by anikdhabal — APPROVED** [review]
Thanks for your work

**Comment by RushikeshGandhmal** [comment]
@hariombalhara the PR is approved, can u please review once.

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -2465,7 +2465,6 @@
   "lowest_rated_members": "Members with lowest rated meetings",
   "csat_score": "CSAT Score",
   "lockedSMS": "Locked SMS",
-  "signing_up_terms": "By proceeding, you agree to our <1>Terms</1> and <2>Privacy Policy</2>.",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/en/common.json:2468]: I thought I left a comment somewhere. Anyway here it goes again, `signing_up_terms` has different values, could you confirm to me you have deleted the correct one.

A screenshot would do. Also, please make sure you restart dev server after the common.json is updated.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left a comment

File: apps/web/public/static/locales/en/common.json (hunk 1 above)
Comment (RushikeshGandhmal) [reply apps/web/public/static/locales/en/common.json:2468]: Sure @hariombalhara , so we deleted correct one now. We get terms as first element in index so keeping message with <0>Terms</0> is correct and removing key with message <1>Terms</1>.

Sorry but screenshot of what ?

Comment (RushikeshGandhmal) [reply apps/web/public/static/locales/en/common.json:2468]: Sure @hariombalhara , so we deleted correct one now. We get terms as first element in array so keeping message with <0>Terms</0> is correct and removing key with message <1>Terms</1>.

Here's video: https://www.loom.com/share/eb3d5f600bcb492e92d31c6b94a9853e
If we keep message with <1>Terms</1> Privacy Policy will not be valid. Only terms will get highlighted.

**Review by hariombalhara — APPROVED** [review]

File: apps/web/public/static/locales/en/common.json (hunk 1 above)
Comment (hariombalhara) [reply apps/web/public/static/locales/en/common.json:2468]: Thanks, great explanation as well !!

Comment (rushikeshrtledgers) [reply apps/web/public/static/locales/en/common.json:2468]: Welcome. Thank you.

Comment (RushikeshGandhmal) [reply apps/web/public/static/locales/en/common.json:2468]: Welcome, Thank you.

**Pull request merged**